tests/includes/API/ApiExceptionTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 5/21/16
+ * Time: 22:35
+ */
+
+use \Waca\API\ApiException;
+
+class ApiExceptionTest extends PHPUnit_Framework_TestCase
+{
+	private $message;
+	private $exception;
+
+	public function setUp() {
+		$this->message = "This is a test ApiException.";
+
+		try {
+			throw new ApiException($this->message);
+		}
+		catch(ApiException $e) {
```

Comment:
Also, general code style comments 
- opening braces go on the next line for methods and classes, but on the same line for control structures
- on this line specifically, a space between the `catch` and the `(` please